hardhat-deploy/test/Cars.test.js

```diff
@@ -0,0 +1,24 @@
+const { ethers, deployments } = require('hardhat');
```

Comment:
... and modify the tests accordingly. so I think the tests that would be needed are:

- assert that first contract is deployed (has non-zero address)
- assert that second contract is deployed as well (has non-zero address)
- assert that the state variable containing the address of the other contract is set correct (address matcheds)